doc/index.rst

```diff
@@ -39,24 +39,28 @@ User Guide
 ==========
 
 The DSM CSC is a support CSC that is used in conjunction with the DSM UI in order
-to produce seeing measurements along the DSM probes line of sight. The functionality to produce these measurements are contained within the UI and are not described in this
-document. The UI is also responsible for the interaction with the DSM probe hardware. 
+to produce seeing measurements along the DSM probes line of sight.
+The functionality to produce these measurements are contained within the UI and are not described in this document.
+The UI is also responsible for the interaction with the DSM probe hardware. 
 The operational interaction of these systems and other details are captured in `SITCOMTN-001 <https://sitcomtn-001.lsst.io/>`_ technical note.
 
-There are two scripts that perform operations of the CSC. They are ``run_dsm.py`` and ``shutdown_dsm.py``. The parameters they take can be found by passing ``-h`` or
-``--help`` to the given script. The ``run_dsm.py`` script constructs the CSC and sends
-it to ``ENABLED`` state, allowing for the CSC to function straight away. It can be
-run in real mode or one of two simulation modes. The simulation modes will be shown
-in the next section. To run the CSC in real mode, do the following.
+There are two scripts that perform operations of the CSC.
+They are ``run_dsm.py`` and ``shutdown_dsm.py``. The parameters they take can be found by passing ``-h`` or ``--help`` to the given script.
+The ``run_dsm.py`` script constructs the CSC and using a flag, sends it to ``ENABLED`` state, allowing for the CSC to function straight away.
+It can be run in real mode or one of two simulation modes.
+The simulation modes will be shown in the next section. To run the CSC in real mode, do the following.
 
 .. prompt:: bash
 
-  run_dsm.py
+  run_dsm.py --state=enabled <index>
+
+The ``<index>`` is an integer value.
+The DSM currently has two indexes (1 and 2) for operational use. 
```

Comment:
Could you please explain why 2 indices? I do not find "For operation use" enlightening.

Are there two dome seeing monitors?

I see that shutdown_dsm.py has a default index, but apparently run_dsm.py does not. This confuses me even more, especially as the help for shutdown_dsm.py says "use the default value unless you sure you know what you are doing"!

Reply:
Yes, there are two field units. I've added a statement to the docs. I've addressed the rest by making the shutdown script more analogous to the way the run script looks with respect to the index and state. I've adjusted the docs to match.